lib/elixir_internal_certificate/accounts/accounts.ex

```diff
@@ -0,0 +1,122 @@
+defmodule ElixirInternalCertificate.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias ElixirInternalCertificate.Accounts.{User, UserToken}
```

Comment:
I think the User and UserToken should be inside the Schemas folder, to follow our structure 

![image](https://user-images.githubusercontent.com/11751745/172757625-dcfd5adc-453d-42e4-923e-b29abd11ba59.png)